.gitignore

```diff
@@ -135,8 +135,8 @@ Keystore/miragexr.keystore
 .DS_Store
 
 # Photon
-/Assets/Photon
-/Assets/Photon.meta
+#/Assets/Photon
```

Comment:
We need to give a warning to Herman, Benedikt (and me), as we have already this component installed and need to remove it before checkout...?

Reply:
before or after, but better before.